activerecord/lib/active_record/delegated_type.rb

```diff
@@ -51,7 +51,7 @@ module ActiveRecord
   #     end
   #   end
   #
-  #   # Schema: messages[ id, subject ]
+  #   # Schema: messages[ id, subject, content ]
```

Comment:
Hey @benoror I think `content` here refers to the `has_rich_text :content`

Reply:
It does, but it was missing from the documented schema, am I right?

Reply:
`has_rich_text :content` stores the content in the `action_text_rich_texts` table, not in the `messages` table.
But I can see it might be confusing, maybe we should remove `content` altogether from the example as we already use `subject` on Message and `content` only adds complexity.

Reply:
yeah, that's why I was confused about it initially, wasn't fully aware how `ActionText` worked to begin with. I believe `content` is an expected attribute of both a `Message` and a `Comment`, how about just getting rid of `has_rich_text` to avoid confusion instead?

Reply:
I think the example is supposed to emphasize that messages and comments are separate tables with separate schemas. Maybe remove `has_rich_text` but use a `body` attribute to `Message`?

Reply:
makes sense, made the change!